civihr_default_theme/scss/civihr_default_theme.style.scss

```diff
@@ -141,3 +141,6 @@
 
 @import "SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/mixins/prefix";
 @import "SCSSROOT/org.civicrm.shoreditch/scss/bootstrap/components/wizard";
+
+// Overrides
+@import "overrides/**/*";
```

Comment:
Should not this be added after line 108?

Reply:
Please check the PR description, i have added the reason its kept in line 146 in point no. 4